.travis.yml

```diff
@@ -15,6 +15,7 @@ services:
 
 php:
   - 7.0
+  - 7.1
```

Comment:
Having multiple jobs on Travis does not make sense. We only need to run tests for the version used by packagist.org